Source/Manager/User/User.swift

```diff
@@ -70,7 +70,15 @@ public class User: UserProtocol {
 
     private let dispatchQueue = DispatchQueue(label: "com.schibsted.identity.User", attributes: [])
     private var _tokens: TokenData?
-    private var isPersistent = false
+    private var _isPersistent = false
+
+    /**
+     Tells you if the user is persistent or not. A persistent user is one where the tokens are
+     stored in the keycchain.
+     */
+    public var isPersistent: Bool {
```

Comment:
Why do you need two versions of the isPersistent variable?

Reply:
I don't :p will fix!